common/web/keyman-version/build.sh

```diff
@@ -24,6 +24,7 @@ builder_describe "Build the include script for current Keyman version" \
   clean \
   build \
   publish \
+  test \
   --dry-run
```

Comment:
Separate thing - I didn't see this option get used anywhere

Reply:
It isn't used (yet) -- but because it's a child builder script, we need the action there in order to run the parent script. It's a little hint to us that we need tests here too :grin: